src/website/layouts/partials/sidenav.pug

```diff
@@ -11,6 +11,34 @@ ul.chi-tabs.-vertical.-lg.-animated
                 if section.disabled
                   //- li
                   //-   a(href=`${rootPath}${section.path}`, disabled="disabled") #{section.title}
+                else if section.accessibilityAccordion
+                  li
+                    .chi-accordion.accessibility-accordion
+                      .chi-accordion__item.-expanded
+                        button.chi-accordion__trigger
+                          .chi-accordion__title #{section.title}
```

Comment:
Style is slightly off here in mobile. Accessibility is rendering as a Level 1 item versus Level 2 on desktop.